test/setup.js

```diff
@@ -0,0 +1,75 @@
+"use strict";
+
+const {createInstrumenter} = require("istanbul-lib-instrument");
+const fs = require("fs");
+const {jsdom, createVirtualConsole} = require("jsdom");
+
+var virtualConsole = createVirtualConsole().sendTo(console);
+
+// Path to the browser-polyfill script, relative to the current work dir
+// where mocha is executed.
+const BROWSER_POLYFILL_PATH = "./dist/browser-polyfill.js";
+
+function setupTestDOMWindow(chromeObject) {
+  return new Promise((resolve, reject) => {
+    let window;
+
+    // If a jsdom window has already been created, reuse it so that
+    // we can retrieve the final code coverage data, which has been
+    // collected in the jsdom window (where the instrumented browser-polyfill
+    // is running).
+    if (global.window) {
+      window = global.window;
+      window.browser = undefined;
```

Comment:
I would prefer to be able to use `delete window.browser`, but unfortunately it didn't worked and I used `window.browser = undefined` as a workaround.

I digged about it and it is a known issue: tmpvar/jsdom#1622

I added an inline comment on it, so that we can remember the "why" of this workaround and when the workaround can be removed.